src/DssSpell.t.sol

```diff
@@ -620,9 +735,19 @@ contract DssSpellTest is DSTest, DSMath {
             (,,, uint256 line, uint256 dust) = vat.ilks(ilk);
             // Convert whole Dai units to expected RAD
             uint256 normalizedTestLine = values.collaterals[ilk].line * RAD;
-            sumlines += values.collaterals[ilk].line;
-            assertEq(line, normalizedTestLine);
-            assertTrue((line >= RAD && line < BILLION * RAD) || line == 0);  // eq 0 or gt eq 1 RAD and lt 1B
+            sumlines += line;
+            (uint256 aL_line, uint256 aL_gap, uint256 aL_ttl,,) = autoLine.ilks(ilk);
```

Comment:
not a big deal right now, but this can be moved inside the conditional

Reply:
Hmmm no, `aL_line` is check in both sides of the conditional and for this spell we want `sumlines` counts all the lines even the ETH-B.

Reply:
yep, I see that now